examples/applications/http-request-processor/README.md

```diff
@@ -0,0 +1,19 @@
+# Http Request processor
```

Comment:
This seems unrelated

Reply:
yes it's based on the http-request branch, Ill put this in draft until we get the other one merged